uint8 msg_cnt
TravelerDataFrame[] data_frames
RegionalExtension[] regional